core/src/main/scala/precog/SbtPrecogBase.scala

```diff
@@ -325,144 +341,152 @@ abstract class SbtPrecogBase extends AutoPlugin {
       unsafeEvictionsFatal := false,
       unsafeEvictionsConf := Seq.empty,
-      unsafeEvictionsCheck / evictionWarningOptions := EvictionWarningOptions.full
+      unsafeEvictionsCheck / evictionWarningOptions := EvictionWarningOptions
+        .full
         .withWarnEvictionSummary(true)
-        .withInfoAllEvictions(false))
+        .withInfoAllEvictions(false)
+    )
 
   override def buildSettings: scala.Seq[Def.Setting[_]] =
     githubActionsSettings ++
-    addCommandAlias("ci", "; checkHeaders; test") ++
-    Seq(
-      organization := "com.precog",
-
-      organizationName := "Precog",
-      organizationHomepage := Some(url("https://precog.com")),
-
-      managedVersions := ManagedVersions(((LocalRootProject / baseDirectory).value / VersionsPath).toPath),
-
-      resolvers := Seq(Resolver.sonatypeRepo("releases")),
-
-      checkLocalEvictions := {
-        if (foundLocalEvictions.nonEmpty) {
-          sys.error(s"found active local evictions: ${foundLocalEvictions.mkString("[", ", ", "]")}; publication is disabled")
-        }
-      },
-
-      trickleDbURI := "https://github.com/precog/build-metadata.git",
-      trickleRepositoryName := Project.normalizeModuleID(uri(trickleRepositoryURI.value).getPath.substring(1)),
-      trickleRepositoryURI := scmInfo.value.map(_.browseUrl).orElse(homepage.value).getOrElse {
-        sys.error("Set 'ThisBuild / trickleRepositoryURI' to the github page of this project")
-      }.toString,
-      trickleGitConfig := {
-        import sbttrickle.git.GitConfig
-        val baseConf = sys.env.get("GITHUB_TOKEN") match {
-          case Some(password) => GitConfig(trickleDbURI.value, "_", password)
-          case _                            => GitConfig(trickleDbURI.value)
-        }
-        if (githubIsWorkflowBuild.value) baseConf
-        else baseConf.withDontPush
-      },
-
-      transferPublishAndTagResources / aggregate := false,
-      transferPublishAndTagResources := {
-        val baseDir = (ThisBuild / baseDirectory).value
-
-        transferScripts(
-          "core",
-          baseDir,
-          "publishAndTag",
-          "readVersion",
-          "isRevision")
-
-        transferToBaseDir(
-          "core",
-          baseDir,
-          "signing-secret.pgp.enc")
-      },
-
-      transferCommonResources / aggregate := false,
-      transferCommonResources := {
-        val baseDir = (ThisBuild / baseDirectory).value
-
-        transferScripts(
-          "core",
-          baseDir,
-          "commonSetup",
-          "discordTravisPost")
-
-        transferToBaseDir("core", baseDir, "common-secrets.yml.enc")
-      },
-
-      secrets := Seq(file("common-secrets.yml.enc")),
-
-      exportSecretsForActions := {
-        val log = streams.value.log
-
-        if (!sys.env.contains("ENCRYPTION_PASSWORD")) {
-          sys.error("$ENCRYPTION_PASSWORD not set")
-        }
+      addCommandAlias("ci", "; headerCheckAll; scalafmtCheckAll; test") ++
```

Comment:
Now using `headerCheckAll` instead of `checkHeaders`. The latter did not fail the build if there is a file without headers.